src/Docs/CLI/Evaluate.hs

```diff
@@ -1046,20 +1046,33 @@ packageModuleUrl (PackageUrl purl) moduleName =
     stripSuffix x s = maybe s reverse $ stripPrefix x $ reverse s
 
 -- =============================
---  HTTP requests
+--  Fetch HTML
 -- =============================
 
+-- | Fetch and cache request's content
 fetchHTML :: HasUrl a => a -> M HtmlPage
 fetchHTML x = do
-  req <- Http.parseRequest (getUrl x)
-  src <- fetch req
+  cache <- State.gets sCache
+  let uri = getUrl x
+  src <- cached cache uri $ fetch uri
```

Comment:
We want to cache HTTP requests because of network and server latency.
When we are fetching stuff from the local computer we can skip the cache altogether. In fact, what caching does is precisely reading files saved locally.

I think we can have `fetch` the way it was before, operating on `Http.Request` and caching the result, then we decide inside of `fetchHTML` whether we will call `fetch` or just read a local file.